ICSSoft.STORMNET.Business/SQLDataService/DbTransactionWrapperAsync.cs

```diff
@@ -0,0 +1,151 @@
+﻿namespace ICSSoft.STORMNET.Business
+{
+    using System;
+    using System.Collections.ObjectModel;
+    using System.Data;
+    using System.Data.Common;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Обёртка над <see cref="DbConnection" /> и <see cref="DbTransaction" />.
+    /// </summary>
+    public class DbTransactionWrapperAsync : IDisposable
+    {
+        private DbTransaction _transaction;
+
+        /// <summary>
+        /// Initializes instance of <see cref="DbTransactionWrapper" />.
```

Comment:
Существует шаблон для комментария к конструктору
```suggestion
        /// Initializes a new instance of the <see cref="DbTransactionWrapper" /> class.
```

Reply:
Сделал по шаблону